Google.Api.Generator.Tests/ProtoTests/ResourceNames/Testing.ResourceNames/ResourceNamesResourceNames.g.cs

```diff
@@ -20,7 +20,7 @@
 
 namespace Testing.ResourceNames
```

Comment:
The fact that this is called ResourceNamesResourceNames.g.cs goes back to my earlier comment about the choice of test names.

Reply:
Yes, will improve naming in a future PR.